### PR TITLE
Fix boskos config path

### PR DESCRIPTION
**Pull request by chizhg**
Boskos config file has been moved, fix its pointers.

/cc @adrcunha 
/cc @chaodaiG

### CODE REVIEW

**Comment by chizhg** [comment]
Hmm, I wonder why these markdown link errors were not detected in the last PR?

File: ci/prow_setup.md [hunk 1]
```diff
@@ -12,13 +12,10 @@
    https://github.com/knative/test-infra/issues/1082.
 
 1. Ensure the GCP projects listed in
```
Comment (adrcunha) [inline ci/prow_setup.md]: I'm making way more changes to this file, including the fixes you have here, in https://github.com/knative/test-infra/pull/1441 you can take this file out of this PR.

Comment (chizhg) [reply ci/prow_setup.md]: Removed.

**Pull request merged**